CTP/README.md

```diff
@@ -97,139 +114,163 @@ This ``Quick Start`` is only for user for reference about how to use ``CTP`` to
+	   ```				
+ 
+	  More parameters setting and parameters explanation within ``shell.conf``, please refer to [CTP/conf/shell.conf](conf/shell.conf)
 
- - Run Tests 
+  - Run Tests 
 	* For **Shell** test:
 	
 	  ```
 	  $ bin/ctp.sh shell -c ./conf/shell.conf
 	  ```   
     
- - Examine the results
-	* Once it completes, you can find the results and logs from ``CTP/result/shell/current_runtime_logs``
-	* ``dispatch_tc_ALL.txt`` will show the total case list, and ``dispatch_tc_FIN_${Node_Name}.txt`` will show the case list which is executed on this server node.
-	* ``main_snapshot.properties`` will save the configurations for your current testing.
-	* ``test_${Node_Name}.log`` will show the logs of testing based on this server node.
+  - Examine the results
+	* When test is completed, you can find the results and logs from ``CTP/result/shell/current_runtime_logs``
+	* ``dispatch_tc_ALL.txt`` will show the total case list, and ``dispatch_tc_FIN_${Node_Name}.txt`` will show the case list which is executed on this instance
+	* ``main_snapshot.properties`` will save all values of parameters configured during testing
+	* ``test_${Node_Name}.log`` will show the logs of testing based on this instance.
 	
 - **Isolation**
   - Prepare
-	* Use one server as controller to checkout CTP, and test node may be one or more, they will be controlled by controller, and CTP must be deployed on each node.
-	* Controller Node configuration is basically same as ``Shell``.
-	  Regarding more parameters for ``isolation`` testing, please refer to [CTP/conf/isolation.conf](conf/isolation.conf)
-	* Environment variables set on test Node:
-	  
-	  ```
-	     JAVA_HOME=$java_installation_directory 
-	     CTP_HOME=$HOME/CTP
-	  ```
-	  
+    * Prepare environments (e.g., one account ``controller`` as controller, another account ``isolation_instance1`` as test instance)
+    * Check out CTP for each account and configure environment variables for CTP ``controller`` and test instance ``isolation_instance1``
+
+      ```
+	  // on controller node
+	  JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)
+
+	  // on test instance node
+	  CTP_HOME (e.g., export CTP_HOME=$HOME/CTP)
+	  JAVA_HOME (e.g., export JAVA_HOME=$HOME/opt/jdk1.6.0_07)
+      ```
+    * **Example** ``isolation.conf`` for test instance, scenario and test build:
+      
+      ```
+	  # Test instance information:
+	  env.instance1.ssh.host=192.168.1.10
+	  env.instance1.ssh.port=22
+	  env.instance1.ssh.user=isolation_instance1
+	  env.instance1.ssh.pwd=12345
+	  env.instance1.broker1.BROKER_PORT=30093
+	  env.instance1.broker2.BROKER_PORT=30094
+      ```
+      ```
+	  # Define the path of test cases used for testing, it should be checked out on test node.
+	  main.testcase.root=/home/qa/isolation_instance1/isolation
+      ```
+      ```
+	  # Define the URL of test build which will be used to test.
+	  # If set 'main.deploy.rebuild_yn' as 'false', this parameter will be ignored.

```

Comment:
"If set 'main.deploy.rebuild_yn' as 'false', this parameter will be ignored."

Please correct it.